nuxt.config.js

```diff
@@ -75,7 +110,7 @@ export default {
   env: {
     // This will be used on the client in axios-baseurl to send requests to the same API endpoing that the server-side
     // code uses.
-    API: process.env.API || 'https://map.restarters.net/map/'
+    API: process.env.API || 'https://map.restarters.net/map/',
```

Comment:
I always thought js/json didn't like hanging commas for some reason - are they OK in some contexts?

Reply:
I think JSON doesn't, but JS allows them and there are good reasons.  

https://time2hack.com/are-you-using-trailing-commas-in-your-javascript/

I think one of the lint checks which come with the latest nuxt has started using them, so they have started appearing in my IDE because of the autofix stuff.